content/en/docs/storage-providers/index-provider.md

```diff
@@ -0,0 +1,217 @@
+   Initialization places IO workload on your storage system. `N` controls the number of deals that are concurrently initialized. See [DagStrore Force Bulk Initialisation](https://lotus.filecoin.io/docs/storage-providers/dagstore/#forcing-bulk-initialization) docs for more information.
+   {{< /alert >}}
+
+   Wait for the initialization to complete. The initialization time is a factor of the volume of data stored since it involves re-indexing the data blocks.
+
+10. **Verify re-creation of DagStore repository**
+
+    The previous step should recreate the DagStore repository, located at `$LOTUS_MARKETS_PATH/dagstore`. Navigate to that directory. Under the subfolder `index`, verify that matching `*.full.idx` files can be found for all files under the same sub-directory in the backup of DagStore taken in step 2.
+
+11. **Announce all indices to the indexers**
+
+    To announce all the indices in bulk to the indexers, run:
+
+    `lotus-miner index announce-all`  if you run a monolith miner process or `lotus-miner --call-on-market index announce-all` if you have split your market subsystem.
+
+    This command generates advertisements and publishes indices onto the indexer gossipsub channel. Look for a series of logs in the markets logs that include `deal announcement sent to index provider`. You should see one such log per deal. The log-line also includes the advertised CID, the deal proposal CID to which it belongs, and the shard key from which its multihash entries are generated. The logs should also include information about the number of multihash entries each advertisement includes. For example:
+
+    ```plaintext
+    deal announcement sent to index provider	{"advertisementCid": "baguqeeqqvr2irdrq45d7npj7elogzpaaam", "shard-key": "baga6ea4seaqegic2h4qoao4urcwhin7tgwlb4cguqymtriheqoyjjaabz6viegq", "proposalCid": "bafyreihhqszkcv3egsb7xkuhyswqjdy3oa2kboi2zjvrbkuj3jgq2g4d4m"}
+    Generated linked chunks of multihashes	{"totalMhCount": 32449, "chunkCount": 2}
+    ```
+
+    Note that the bulk advertisement only announces deals that are not expired and handed over to the sealing subsystem. The daemon will not advertise the expired deals. For any remaining deals, the advertisement will occur after they are given to the sealing subsystem.
+
+    Wait for the bulk indexing announcement to complete. The bulk announcement is complete when `finished announcing active deals to index provider` is logged.
+
+## Bulk Index Announcement
+
+You can use the following CLI command to bulk-announce all deals made by the storage provider:
+
+```shell
+lotus-miner index announce-all
+```
+
+{{< alert icon="tip" >}}
+Note that if you have split your markets subsystem, you will need to specify `--call-on-market`.
+{{< /alert >}}
+
+## Configuration
+
+You can adjust the values under the `IndexProvider` session in the `config.toml` of your market process to configure indexes' announcement to the indexer.
+
+If the session doesn't exist, you can manually add it:
```

Comment:
`s/session/section`?

Reply:
Thank you for pointing this out.

My spelling mistakes will outlive me; I am sure.

Reply:
Fix added here: https://github.com/filecoin-project/lotus-docs/pull/94